src/Options/ContactDetails.php

```diff
@@ -0,0 +1,137 @@
+<?php namespace Leean\Elements\Options;
+
+/**
+ * Class ContactDetails.
+ *
+ * @package Leean\Elements\Options
+ */
+class ContactDetails
```

Comment:
Do you think a menu fits in this wp-elements module?
It would have to be a LeanMenu instead.
However, we can still reuse this wp-elements for other things.